packages/react-components/src/components/MessageThread.tsx

```diff
@@ -411,6 +411,11 @@ const memoizeAllMessages = memoizeFnAll(
                   : {}
             }}
             body={{ className: chatMessageRenderStyles.body }}
+            statusIcon={
+              <div className={mergeStyles({ paddingLeft: '0.25rem' })}>
```

Comment:
could you add the missed custom styles
`styles?.messageStatusContainer ? styles.messageStatusContainer(message.mine ?? false) : ''`